.changeset/gold-shirts-switch.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Let's be mindful of changeset granuality. This change should be split across 2 files. The message you wrote is relevant only for Studio